tests/goald_configuration/test_model.py

```diff
@@ -53,7 +53,8 @@ def test_deployment():
 
 
 def test_goal():
-    goal = Goal(
-        identification='g1'
-    )
-    assert goal.identification == 'g1'
+    goal = Goal()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'goal' is assigned to but never used](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4861978)